src/js/components/DataTable/Body.js

```diff
@@ -181,20 +178,25 @@ const Body = forwardRef(
 
     return (
       <Keyboard
-        onEnter={
-          // active is undefined if user never used the keyboard
-          onClickRow &&
-          active >= 0 &&
-          (!disabled ||
-            !disabled.includes(datumValue(data[active], primaryProperty)))
-            ? (event) => {
-                event.persist();
-                const adjustedEvent = event;
-                adjustedEvent.datum = data[active];
-                onClickRow(adjustedEvent);
-              }
-            : undefined
-        }
+        onEnter={(event) => {
+          if (
+            (onClickRow && active >= 0 && !disabled) ||
```

Comment:
Formatting seems to have changed here. It should be:

```
(onClickRow && active >= 0 && (!disabled || !disabled.includes(datumValue(data[active], primaryProperty))

Reply:
fixed